dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -348,6 +363,53 @@
 		</ItemGroup>
 	</Target>
 
+	<!-- App bundle creation tasks -->
+	<Target Name="_AOTCompile"
+			Condition="'$(_SdkIsSimulator)' != 'true' And '$(_PlatformName)' != 'macOS'"
```

Comment:
XM supports AOT
Better add a note this is not covered (yet) in the commit description